kalite/topic_tools/content_models.py

```diff
@@ -281,7 +286,9 @@ def get_topic_update_nodes(parent=None, **kwargs):
             Item.total_files,
             Item.id,
             Item.path,
-        ).join(Parent, on=(Item.parent == Parent.pk)).where((selector) & (Item.total_files != 0))
+        ).join(Parent, on=(Item.parent == Parent.pk)).where((selector))
+        # Not filtering on total_files != 0
+        # ).join(Parent, on=(Item.parent == Parent.pk)).where((selector) & (Item.total_files != 0))
```

Comment:
This needs to be here, otherwise exercises will be included in the download interface.

In the database, do all topics still have total_files == 0?